### DISCUSSION TITLE
Upload ssh key issue

### DISCUSSION THREAD

**Discussion by netbja** [roxy-wi/roxy-wi#351] [Q&A] [answered]
Dear all,

roxy-wi : v6.2.1.0

Trying to upload ssh key on servers, generate a error message. 

`error: Cannot save SSH key file: module 'paramiko.pkey' has no attribute 'load_private_key' 
`
And no key was present in my /var/lib/roxy-wi/keys/ folder
The test was made with : ecdsa, rsa and pem format.
Any idea ?
Thanks.

**Comment by Aidaho12** [comment, marked as answer]
Hi,
Try to install paramiko-ng module via pip

**Reply by netbja** [reply]
Sorry, in my installation, I went through the paramiko-ng package !
It's working fine now !
Thanks.
